File: Python/3-Basic-Actions/7-Send_keys.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys

import pdb

driver = webdriver.Chrome()

url = "https://www.google.com"

try:
    driver.get(url)
    search_field = driver.find_element('name', 'q')
    search_field.send_keys('Selenium Webdriver')
    pdb.set_trace()  # Enter 'c' to continue

    search_field.send_keys(Keys.DOWN)
    search_field.send_keys(Keys.DOWN)
    pdb.set_trace()  # Enter 'c' to continue

    search_field.send_keys(Keys.ENTER)
    pdb.set_trace()  # Enter 'c' to continue
finally:
    driver.quit()

File: Javascript/3-Basic-Actions/7-Send-Keys.js
const { Builder, By, Key } = require('selenium-webdriver');

(async function(){
    let driver = await new Builder().forBrowser('chrome').build();
    try{
        await driver.get('https://www.google.com/');
        /**
         * Go go google.com and search for "Selenium Webdriver"
         */
        const my_element = await driver.findElement(By.name('q'));
        await my_element.sendKeys("Selenium Webdriver");
        await my_element.sendKeys(Key.DOWN);
        await my_element.sendKeys(Key.DOWN);
        await my_element.sendKeys(Key.ENTER);
    } finally {
        await driver.quit()
    }
})()